controllers/discovery/initDiscoveryUserData.js

```diff
@@ -69,11 +88,21 @@ const InitDiscoveryUserData = async (req, res) => {
     });
     let suggestedUsers = usersWithCommonFollowerResult;
 
+    let totalData = await sequelize.query(totalDataQuery, {
+      type: QueryTypes.SELECT,
+      replacements: {
+        userId
+      },
+      raw: true
+    });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the input parameters for the SQL query should be validated and sanitized to prevent potential SQL injection attacks. 



<!-- This is an auto-generated comment by OSS CodeRabbit -->